InclusionBot.md

```diff
@@ -73,9 +73,11 @@ approves your changes, they will be merged and become active in the bot.
-| mumbo jumbo                                      | jargon, confusing language                                                                    | The term "mumbo jumbo" was used to stereotype African languages and cultures that didn't make sense to European colonizers. <br><br> See [Here are some commonly used terms that actually have racist origins](https://abcnews.go.com/Politics/commonly-terms-racist-origins/story?id=71840410) for more.                                                                                                                                                                                                                                                                                                                                                            |
+| Goddamn, goddam, goddamnit, goddammit            | awful<br>terrible<br>wretched                                                                 | This term can be offensive or triggering to some religious traditions because it implies a deity cursing or reproaching people. We want to show respect towards all religious traditions.                                                                                                                                                                                                                              |
+| jesus fucking, fucking christ                    | Wow<br>oh no<br>oh my                                                                         | We want to show respect towards all religious traditions.                                                                                                                                                                                                                                                                                                                                                              |
+| mumbo jumbo                                      | jargon<br>confusing language                                                                  | The term ":TERM:" was used to stereotype African languages and cultures that didn't make sense to European colonizers.<br><br>See [some other commonly used terms that have racist origins](https://abcnews.go.com/Politics/commonly-terms-racist-origins/story?id=71840410).                                                                                                                                          |
+
+| kowtow, kow tow, cowtow | submit<br>grovel | The modern usage of this word refers to groveling or submission, which is a disconnect from its real meaning as an act of deep respect. Modern usage of this term misses cultural nuance and perpetuates cultural misunderstanding.<br><br>See [the Slack thread](https://gsa-tts.slack.com/archives/C10EGE62W/p1727210606818389) where teammates discuss the term for more information.|
```

Comment:
When I look at the rendered file (scroll slightly up from [this anchor](https://github.com/18F/charlie/blob/a8bd302f3513f7a8b91b0f3b55048d0dbeab96a5/InclusionBot.md#sexist)), this line of the table isn't rendering.

I'm seeing a newline at 79, I bet that's the culprit.